libs/novui/src/panda-sandbox/recipes/sva.stories.tsx

```diff
@@ -0,0 +1,111 @@
+import { Meta, StoryFn } from '@storybook/react/*';
+import React, { FC } from 'react';
+import { RecipeVariant, sva } from '../../../styled-system/css';
+import { Stack } from '../../../styled-system/jsx';
+
+export default {
+  title: 'Panda Sandbox/recipes/sva',
+} as Meta;
+
+/**
+ * sva is a function to create recipes that target specific named parts / elements ("slots")
+ * of a component -- it returns an object of classNames.
+ * https://panda-css.com/docs/concepts/slot-recipes#atomic-slot-recipe-or-sva
```

Comment:
This is great, however I'd consider using a more familiar form input example that everyone will have seen before. Then, I'd add something like:

"sva is similar to cva, but is typically used when we have a composition of components (i.e. a composite component) that contains multiple atomic components that we would like to style independently. A good example of where we would use sva is for a form input component recipe, where we have a label, text input, and helper text that each have their own styles."